webapp/src/containers/MenuContainer.js

```diff
@@ -22,7 +22,7 @@ class MenuContainer extends React.Component {
   }
 
   changeMenuVisibility(menuVisible) {
-    (menuVisible) ? this.props.openMenu() : this.props.closeMenu()
+    menuVisible ? this.props.openMenu() : this.props.closeMenu()
```

Comment:
I know this change wasn't introduced here, but to me this is a bit overly complicated.

We have an if condition below that already checks for the `menuVisible` variable, so I think it's easier to read if we do something like:

```javascript
    if (menuVisible) {
      this.props.openMenu()

      // Wait a bit for the Menu animation to end
      setTimeout(() => this.props.fetchOngoingAuctions(), 500)
    } else {
      this.props.closeMenu()
    }
```

What do you think?

Reply:
Done!